tests/examples/source/devfiles/nodejs/devfile-with-no-default.yaml

```diff
@@ -0,0 +1,47 @@
+schemaVersion: 2.0.0
+metadata:
+  name: nodejs
+starterProjects:
+  - name: nodejs-starter
+    git:
+      remotes:
+        origin: "https://github.com/odo-devfiles/nodejs-ex.git"
+components:
+  - name: runtime
+    container:
+      image: registry.access.redhat.com/ubi8/nodejs-12:1-36
+      memoryLimit: 1024Mi
+      endpoints:
+        - name: "3000-tcp"
+          targetPort: 3000
+      mountSources: true
+commands:
+  - id: firstbuild
+    exec:
+      component: runtime
+      commandLine: npm install
+      workingDir: ${PROJECTS_ROOT}
+      group:
+        kind: build
+        isDefault: true
```

Comment:
File name here is `devfile-with-no-default.yaml` but there's a `isDefault: true` here. Or is this devfile about no default `run` command?

Reply:
Yes, it is about no default run command.